types/camunda-external-task-client-js/index.d.ts

```diff
@@ -47,6 +47,7 @@ export class Variables {
     getAll(): ValueMap;
     getAllTyped(): TypedValueMap;
     set(variableName: string, value: Value): Variables;
+    setTransient(variableName: string, value: Value): Variables;
```

Comment:
It appears this method was added in version 2.2 of this package. Could you update the package version in the header of this file to reflect that? Right now it says `1.3`, and in `1.3` `setTransient` wasn't yet present. Thanks!